app/User.php

```diff
@@ -43,12 +44,25 @@ public function lessons()
         return $this->hasMany(Lesson::class);
     }
 
+    public function followersCount() 
+    {
+        $followers = Connection::where('following_id', $this->id)->get();
+
+        return count($followers);
```

Comment:
instead of this one, you can call it directly since user have a relationship with connects model you can declare it like this
```
public function followerCount()
{
    return $this->connections->count();
}
```

lesser and more direct.